src/code42cli/util.py

```diff
@@ -69,10 +69,13 @@ def find_format_width(record, header):
 
 def format_to_table(rows, column_size):
     """Prints result in left justified format in a tabular form."""
```

Comment:
This comment needs to be changed, as it no longer prints the results but just returns them instead

Reply:
Changed.